static/src/javascripts/projects/common/modules/experiments/tests/participation-star-ratings.js

```diff
@@ -45,7 +47,10 @@ define([
 
         this.canRun = function () {
             // Commentable, Film reviews
-            return config.page.section === 'film' && config.page.toneIds === 'tone/reviews' && config.page.commentable;
+            return config.page.section === 'film' &&
+                config.page.toneIds === 'tone/reviews' &&
+                config.page.commentable &&
```

Comment:
I think because the participation baseline is comparison with commentable articles, so only with this condition do we have a valid comparison point.